lib/rouge/lexers/sas.rb

```diff
@@ -0,0 +1,568 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class SAS < RegexLexer
+      title "SAS"
+      desc "SAS (Statistical Analysis Software)"
+      tag 'sas'
+      filenames '*.sas'
+      mimetypes *%w(
```

Comment:
We're taking a look at reducing the resource usage of Rouge. To that end, could you pass string literals to `mimetypes`?

Reply:
Thanks; done in https://github.com/rouge-ruby/rouge/pull/1107/commits/3191ee40b262959258188a1c4661249f0daf3520.